src/utils/puns.ts

```diff
@@ -13,7 +13,12 @@ const CLEAN_RUN_PUNS = [
   'Fin-tastic job!',
   'You are dolphinitely doing great!',
   "It doesn't get any betta than this!",
-  "You're piranha roll now!"
+  "You're piranha roll now!",
+  "Valedictorian of the School of FSH",
+  "If you're FSHing for compliments, you got it!",
```

Comment:
Unfortunately, puns must be 36 characters or less (else they don't fit in the box).  I should probably put in a unit test that enforces this.

Reply:
DIdn't know that.

Reply:
Yeah, I _thought_ I had put a comment about this at the top of the file, but apparently I thought wrong.